library.properties

```diff
@@ -0,0 +1,10 @@
+name=RESTuino
+version=0.0.1
+author=Yuki Takeyama
+maintainer=takeyamayuki <***@example.com>
+sentence=A firmware for ESP32 to handle arduino GPIO via REST API
```

Comment:
Would `A library to handle arduino GPIO via REST API` be better for the sentence, paragraph?